### PR TITLE
fix(pou): Fix landscape camera mode

### PR DESCRIPTION
**Pull request by Likita**
- Fix landscape camera mode
- Adding Dashboard button in case of error after signing

### CODE REVIEW

File: packages/apps/proof-of-us/src/components/Status/ErrorStatus.tsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 import { Stack } from '@kadena/react-ui';
+import Link from 'next/link';
 import type { FC, PropsWithChildren } from 'react';
 import { Button } from '../Button/Button';
 import { MessageBlock } from '../MessageBlock/MessageBlock';
 import { Heading } from '../Typography/Heading';
 
 interface IProps extends PropsWithChildren {
-  handleClose?: () => void;
+  handleClose?: string;
```
Comment (sstraatemans) [inline packages/apps/proof-of-us/src/components/Status/ErrorStatus.tsx:10]: if we make it a url, maybe better for readability to rename.
`handleClose` kind of implies a function.
maybe `closeUrl` or something

**Review by sstraatemans — CHANGES_REQUESTED** [review]
some small remarks

can we make it a bit bigger maybe? feels small? I understand that the title maybe goes over the view.
if that is the case. maybe we should hide the title on landscape mode. camerview is more important.

**Comment by Ghislain89** [comment]
Tested it, seems to work fine!

**Review by sstraatemans — APPROVED** [review]
if ghislain is happy. Im happy

**Pull request merged**